cms/src/components/Input/Input.styles.ts

```diff
@@ -60,6 +66,14 @@ export const InputWrapper = styled.div<{ checkIsValid: boolean; begIcon: boolean
           } \
         }'}
 
+  ${({ isError }) =>
+    isError &&
+    'outline: 2px solid #dc0c31; \
+    & + label#support-label { \
+      visibility: visible; \
+      color: #dc0c31; \
+    }'}     
+
   &:has(input:focus) {
     outline: 2px solid #939393;
     & + label#support-label {
```

Comment:
is the code working properly?
![Screenshot 2023-08-28 at 11 33 54](https://github.com/baza-trainee/feeda/assets/90103151/fc16b124-8ef8-4531-8eb0-a36d090bd0a4)